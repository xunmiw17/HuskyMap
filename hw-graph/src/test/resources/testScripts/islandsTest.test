# Create a graph
CreateGraph graph1

# Island 1
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add multiple edges in island 1
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2
AddEdge graph1 n3 n2 e3

# Island 2
AddNode graph1 n4

# Add a self edge in the node in island 2
AddEdge graph1 n4 n4 e4

# Island 3
AddNode graph1 n5
AddNode graph1 n6

# Add multiple edges in island 3
AddEdge graph1 n5 n6 e5
AddEdge graph1 n5 n6 e6
AddEdge graph1 n5 n6 e7
AddEdge graph1 n6 n5 e8

# Print the nodes in the graph and the outgoing edges from all six nodes
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3
ListChildren graph1 n4
ListChildren graph1 n5
ListChildren graph1 n6