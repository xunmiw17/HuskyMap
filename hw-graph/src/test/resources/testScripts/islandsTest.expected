# Create a graph
created graph graph1

# Island 1
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add multiple edges in island 1
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1
added edge e3 from n3 to n2 in graph1

# Island 2
added node n4 to graph1

# Add a self edge in the node in island 2
added edge e4 from n4 to n4 in graph1

# Island 3
added node n5 to graph1
added node n6 to graph1

# Add multiple edges in island 3
added edge e5 from n5 to n6 in graph1
added edge e6 from n5 to n6 in graph1
added edge e7 from n5 to n6 in graph1
added edge e8 from n6 to n5 in graph1

# Print the nodes in the graph and the outgoing edges from all six nodes
graph1 contains: n1 n2 n3 n4 n5 n6
the children of n1 in graph1 are: n2(e1) n3(e2)
the children of n2 in graph1 are:
the children of n3 in graph1 are: n2(e3)
the children of n4 in graph1 are: n4(e4)
the children of n5 in graph1 are: n6(e5) n6(e6) n6(e7)
the children of n6 in graph1 are: n5(e8)